docs/mockups/benchmark-density/shared.css

```
@@ -0,0 +1,661 @@
+.tone-sand { --tone: #fbbf24; --tone-soft: #392d14; }
+.tone-cyan { --tone: #22d3ee; --tone-soft: #12323a; }
+.tone-rose { --tone: #fb7185; --tone-soft: #3f1e25; }
+
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)